clients/runtime/src/tests.rs

```diff
@@ -106,22 +109,52 @@ async fn test_subxt_processing_events_after_dispatch_error() {
 	result.1.unwrap();
 }
 
-#[tokio::test(flavor = "multi_thread")]
+#[tokio::test(flavor = "multi_thread", worker_threads = 4)]
 async fn test_register_vault() {
-	let is_public_network = false;
-	let (client, _tmp_dir) =
-		default_provider_client(AccountKeyring::Alice, is_public_network).await;
-	let parachain_rpc = setup_provider(client.clone(), AccountKeyring::Alice).await;
-	set_exchange_rate(client.clone()).await;
-
-	let vault_id = VaultId::new(
-		AccountKeyring::Alice.into(),
-		DEFAULT_TESTING_CURRENCY,
-		DEFAULT_WRAPPED_CURRENCY,
-	);
-
-	parachain_rpc.register_public_key(dummy_public_key()).await.unwrap();
-	parachain_rpc.register_vault(&vault_id, 3 * 10u128.pow(12)).await.unwrap();
-	parachain_rpc.get_vault(&vault_id).await.unwrap();
-	assert_eq!(parachain_rpc.get_public_key().await.unwrap(), Some(dummy_public_key()));
+    env_logger::init_from_env(
```

Comment:
This code change (this and the following lines) seems to go beyond upgrading to v1.1.0, why is that?

Reply:
Also temporary.